build/build-sdk-images/node/gen.sh

```diff
@@ -25,8 +25,9 @@ googleapis=/go/src/agones.dev/agones/proto/googleapis
 
 cd /go/src/agones.dev/agones
 
-protoc -I ${googleapis} -I ${sdk} --grpc_out=minimum_node_version=12:./sdks/nodejs/lib --plugin=protoc-gen-grpc=`which grpc_node_plugin` sdk.proto
-protoc -I ${googleapis} -I ${sdk} --js_out=import_style=commonjs,binary:./sdks/nodejs/lib sdk.proto ${googleapis}/google/api/annotations.proto ${googleapis}/google/api/http.proto
+npm install --unsafe-perm --global grpc-tools
```

Comment:
Curious - do you think we should bake grpc-tools into the [Dockerfile](https://github.com/googleforgames/agones/blob/3b949992e0f023562a4fa7cd9e019edfccb49c0b/build/build-sdk-images/node/Dockerfile) ?

Reply:
Also, I think it would be good to lock this to a version, to avoid any weirdness between runs.

Reply:
I started out this way and had some issues but now it's working `gen.sh` then will take a look at moving it. Will add the latest version too.

Another way of doing this is to add to the package.json of the sdk and have a build/pre-build step there. One advantage of this is that a dependency bot could keep it up to date (with other dependencies), also the SDK would be self contained and it's clear from the package.json which version of grpc-tools was used. We would then update gen.sh to just run `npm run build` or similar.

This is just an idea though and would not do as part of this code review :)